src/Item.php

```diff
@@ -905,7 +905,7 @@ public function get_links($rel = 'alternate')
      * @param int $key The enclosure that you want to return.  Remember that arrays begin with 0, not 1
      * @return \SimplePie\Enclosure|null
      */
-    public function get_enclosure($key = 0, $prefer = null)
+    public function get_enclosure(int $key = 0, $prefer = null)
```

Comment:
`$prefer` is a weird argument introduced in 4bbd7eacfb6940d06899fa74a08754389ddd98de, probably can be safely removed, it has never been used according to `git log -p -G 'get_enclosure\b.+,'` and `git log -p -G '\$prefer'`.